stratum/p4c_backends/fpm/annotation_mapper.cc

```diff
@@ -216,5 +221,53 @@ bool AnnotationMapper::MapTableAnnotation(
   return true;
 }
 
+bool AnnotationMapper::HandleActionAnnotations(const std::string& action_name,
+                            const hal::P4InfoManager& p4_info_manager,
+                            hal::P4ActionDescriptor* action_descriptor) {
+  // The name is always the first annotation lookup.
+  bool name_ok = MapActionAnnotation(action_name, action_descriptor);
```

Comment:
any reason we need to create `name_ok` variable?

Reply:
No particular reason, `HandleFieldAnnotations` looks the same: https://github.com/stratum/stratum/pull/20/files/1372bd31c16718c27818e6763951a5536d6ab4f3#diff-791e4f5f37a5882c64543f76fac21897R144-R149